examples/with-wallet-stamper/src/components/turnkey-provider.tsx

```diff
@@ -101,13 +98,7 @@ export const TurnkeyProvider: React.FC<TurnkeyProviderProps> = ({
     chainType: ChainType = ChainType.SOLANA
   ) {
     setAuthenticating(true);
-    let publicKey = null;
-    if (chainType === ChainType.SOLANA) {
-      const solanaWallet = wallet as SolanaWalletInterface;
-      publicKey = solanaWallet.recoverPublicKey();
-    } else if (chainType === ChainType.EVM) {
-      const evmWallet = wallet as EvmWalletInterface;
-    }
+    const publicKey = await wallet?.getPublicKey();
```

Comment:
Reflects change to the wallet stamper interface, we now use `getPublicKey` for both EVM and Solana.